doc/source/c-api.rst

```diff
@@ -164,6 +164,10 @@ Array
     :project: TileDB-C
 .. doxygenfunction:: tiledb_array_open_with_key
     :project: TileDB-C
+.. doxygenfunction:: tiledb_array_open_at
+    :project: TileDB-C
+.. doxygenfunction:: tiledb_array_open_with_key_at
```

Comment:
It might be better to name it `tiledb_array_open_at_with_key` to keep the verb "open at" together. Also all the other `with_key` functions end with `with_key`.

Reply:
That's fine, I'll do the change.